frontend/packages/api_client/src/apis/V1Api.ts

```diff
@@ -227,14 +232,58 @@ export class V1Api extends runtime.BaseAPI {
     }
 
     /**
-     * Uses InstanceReference to iterate all types of instances, and serializes them.  The fields that are returned for each instance depend on its instance_type and whether you are listing all instances (returns fewer fields) or just one instance (returns all fields).  The only fields that are available for all instances, regardless of type, are the fields defined on the InstanceReference class, namely:  * `id` * `name` * `created` * `modified` * `instance_type` * `is_archived`  Note that IDs used for instances are always the ID of the InstanceReference object, which may not be the same as the ID of the specific Instance subclass (e.g. the OpenEdXInstance object has its own ID which should never be used - just use its InstanceReference ID). This detail is managed by the API so users of the API should not generally need to be aware of it.
+     * Uses InstanceReference to iterate all types of instances, and serializes them.  The fields that are returned for each instance depend on its instance_type and whether you are listing all instances (returns fewer fields) or just one instance (returns all fields).  The only fields that are available for all instances, regardless of type, are the fields defined on the InstanceReference class, namely:  * `id` * `name` * `notes` * `created` * `modified` * `instance_type` * `is_archived`  Note that IDs used for instances are always the ID of the InstanceReference object, which may not be the same as the ID of the specific Instance subclass (e.g. the OpenEdXInstance object has its own ID which should never be used - just use its InstanceReference ID). This detail is managed by the API so users of the API should not generally need to be aware of it.
      * API to list and manipulate instances.
      */
     async instanceRead(requestParameters: InstanceReadRequest): Promise<InstanceReferenceDetailed> {
         const response = await this.instanceReadRaw(requestParameters);
         return await response.value();
     }
 
+    /**
```

Comment:
@gabor-boros The _instance notes_  parts of the code (changing notes through a new API) don't seem directly related to this PR. Could you mention in the description why and how this is being changed? (and how to test it)? (or maybe remove this part of the code)

Reply:
@clemente The js/ts client is updated (`npm run update-api-client`), hence some changes which may not be related completely to this change. (Also added this to the description).

Reply:
Yep looks like this regen was missed when we added the instance notes api endpoint. :+1: